assets/src/blocks/HubspotForm/HubspotFormEditor.js

```diff
@@ -0,0 +1,161 @@
+import { Fragment, useEffect, useRef } from '@wordpress/element';
+import { MediaUpload, MediaUploadCheck, RichText } from '@wordpress/block-editor';
+import { Button } from '@wordpress/components';
+import { Sidebar } from './HubspotFormSidebar';
+import { useToAttribute } from './hooks/useToAttribute';
+import { useBackgroundImage } from './hooks/useBackgroundImage';
+import { getStyleFromClassName } from '../getStyleFromClassName';
+
+const { __ } = wp.i18n;
+
+export const HubspotFormEditor = ({
+  attributes: {
+    block_background_image_id,
+    block_background_image_url,
+    block_text,
+    block_title,
+    block_style,
+    cta_text,
+    cta_link,
+    cta_new_tab,
+    form_text,
+    form_title,
+    hubspot_shortcode,
+    hubspot_thankyou_message,
+    enable_custom_hubspot_thankyou_message,
+    className,
+  },
+  setAttributes,
+}) => {
+  const wrapperRef = useRef(null);
```

Comment:
This ref now seems unused, can probably be removed.

Reply:
It's used [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/641/files/02445295ca2c4f40f40f94e456928e42cccd957a#diff-74590d4ebb9c6d57f60a788911755629e1110b04dd2b87c5a18c4c963f26c261R83)

Reply:
Well the reference is passed there, but nothing actually uses it, right?

Reply:
Yes, I see your point and you're completely right. It was used by a removed hook. Good catch!